profiles/rawls-sa/terraform/rawls/variables.tf.ctmpl

```diff
@@ -5,19 +5,34 @@ variable "google_project" {
   default = "{{env "GOOGLE_PROJECT"}}"
   description = "The google project as specified in the application json"
 }
+
+variable "deployment_manager_google_project" {
+  default = "{{if (env "DEPLOYMENT_MANAGER_PROJECT")}}{{env "DEPLOYMENT_MANAGER_PROJECT"}}{{else}}{{env "GOOGLE_PROJECT"}}-deployment-mgr{{end}}"
```

Comment:
can we change this to be `terra-deployments-{envname}`

Reply:
done